sources/osg/LightSource.js

```diff
@@ -18,7 +21,25 @@ LightSource.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit(
     },
     setLight: function ( light ) {
         this._light = light;
+    },
+    setReferenceFrame: function ( value ) {
+        this._referenceFrame = value;
+    },
+    getReferenceFrame: function () {
+        return this._referenceFrame;
+    },
+    computeBound: function ( bsphere ) {
+        Node.prototype.computeBound.call( this, bsphere );
+        if ( this._light !== undefined && this._referenceFrame === TransformEnums.RELATIVE_RF ) {
+            var position = this._light.getPosition();
+            if ( position[ 3 ] === 0.0 ) {
+                var div = 1.0 / position[ 3 ];
```

Comment:
isn't it a divide By Zero there?
I'm sorry, I  don't get what that code part should do ?
maybe Light does have impact on bounding box to prevent frustum culling, but why ligth position is "scaled"?